providers/redfish/redfish.go

```diff
@@ -26,17 +26,35 @@ var (
 	Features = registrar.Features{
 		providers.FeaturePowerSet,
 		providers.FeaturePowerState,
+		providers.FeaturePowersensors,
```

Comment:
might do `providers.FeaturePowerSensors,` instead

Reply:
thanks, fixed in 3c3eec6